content/en/docs/instrumentation/java/extensibility.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Extensibility
+description: Extensions add new features and capabilities to the agent without having to create a separate distribution or to fork this repository. For example, you can create custom samplers or span exporters, set new defaults, and embed it all in the agent to obtain a single jar file.
```

Comment:
This page is about extensions, so why not use that as a title?

> title: Extensions

Also, the description should be short, for example:

> What is an extensions and when should I create one

Reply:
If you change the title, also rename the file.

Reply:
done